eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -1364,6 +1364,15 @@ public function listTrashed($offset, $limit, array $sort = null)
         return $rows;
     }
 
+    public function countTrashed(): int
+    {
+        $query = $this->connection->createQueryBuilder()
+            ->select('COUNT(node_id) as count')
```

Comment:
Perhaps we should use here something like `$this->connection->getDatabasePlatform()->getCountExpression('node_id')`

Reply:
Nice one, I was not aware of it 👍

Reply:
you imply this here right?


```suggestion
            ->select($this->connection->getDatabasePlatform()->getCountExpression('node_id'))
```

Reply:
yes, this is what i mean